cumulus/polkadot-parachain/src/chain_spec/bridge_hubs.rs

```diff
@@ -273,109 +237,7 @@ pub mod rococo {
 
 /// Sub-module for Kusama setup
 pub mod kusama {
-	use super::{BridgeHubBalance, ParaId};
-	use crate::chain_spec::{
-		get_account_id_from_seed, get_collator_keys_from_seed, Extensions, GenericChainSpec,
-		SAFE_XCM_VERSION,
-	};
-	use parachains_common::{AccountId, AuraId};
-	use sc_chain_spec::ChainType;
-	use sp_core::sr25519;
-
 	pub(crate) const BRIDGE_HUB_KUSAMA: &str = "bridge-hub-kusama";
```

Comment:
Just curious why this stays?

Reply:
no needs, everything is/will be soon in fellows repo

Reply:
This and the five others remain for running `polkadot-parachain` for these runtimes, even if the chainspecs and code lives elsewhere. This does look a bit lonely in this file though I agree, the others are directly in the `command.rs` except for bridge hubs.

Reply:
aha, ok, thank you, and will they stay here forever or it is just temp solution?

Reply:
I think it's something to be discussed, but the recent changes allow something like running with a supplied chainspec and wasm blob as arguments passed (as paths) to `polkadot-parachain` instead of these hardcoded values. That way we also drop the dependency on the testnet code for `polkadot-parachain`. The chainspecs could be generated by a different binary, even for the testnets. WDYT?
I think it aligns well with the 'do one thing and do it well' concept for a program as well as improving compile time and removing dependencies from `polkadot-parachain`.

Reply:
Sorry, to answer your question explicitly: in my mind it's a temp solution before we come to a consensus about what to do in the long run.